manageability-docker/TBH/Release-README

```diff
@@ -0,0 +1,24 @@
+QUICK README
+=============
+1.  To install both TC and BC run the install-inb.sh script with root privileges. Before running the script update the following fields in azure_conf_file with the necessary information to have device connected to cloud.
+        - SCOPE_ID <refer Azure user guide documentation on how to fetch SCOPE_ID>
```

Comment:
Actually we need to just point them to the Install guides in our /doc directory.  Did you find this somewhere else in our code?  We need to update that if so.